src/v2/Components/LoginSignUpBanner/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import { Box, color, BoxProps } from "@artsy/palette"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import styled from "styled-components"
+
+interface Props extends BoxProps {
+  href: string
+}
+
+const Button = styled(Box).attrs({
+  flexGrow: 1,
+  py: "8px",
+})`
+  width: 50%;
+  height: 40px;
+  text-align: center;
+  background-color: ${color("black80")};
+  color: ${color("white100")};
+  font-family: "ll-unica77", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-weight: bold;
+  font-size: 15px;
+`
```

Comment:
It seems to me that we should just adapt our `Button` component to support `as` — we should never be setting font-family/font-size etc directly.

Reply:
What do you mean by that? Could you give some example?

Reply:
https://github.com/artsy/palette/pull/946

So we can pass `as` with either a string name of an element or an actual component and the component will internally use that as the basis while keeping all the styling. So once this is released we can just do `<Button as={RouterLink} to="/log_in">` for example.

```tsx
<Flex p={1}>
  <Button flex={1} as={RouterLink} to="/sign>Sign up</Button>

  <Spacer ml={1} />

  <Button flex={1}>Log in</Button>
</Flex>

<Separator />
```

Chatting about the color scheme inconsistencies here: https://artsy.slack.com/archives/C9XJKPY9W/p1623078390072600

Reply:
I've got a PR coming for this as I need to move it anyway for an unrelated reason!

Reply:
Thank you for clarifying! So as I understand you are fixed this? Or I should do it?

Reply:
All good now!